ui/imports/shared/views/NetworksSimpleRoutingView.qml

```diff
@@ -114,7 +116,8 @@ RowLayout {
                 rightPadding: 5
                 implicitWidth: 150
                 title: chainName
-                subTitle: root.weiToEth(balance)
+                subTitle: selectedAccount && selectedAccount!== undefined && selectedAsset!== undefined ?
```

Comment:
This should read `typeof selectedAccount !== undefined`

Reply:
In other places too

Reply:
selectedAccount!== undefined works too, is this better ? @caybro

Reply:
Ok then, thanks for verifying

Reply:
approve please? :)